storybook/src/components/Alert/Alert.stories.tsx

```diff
@@ -24,17 +24,19 @@ export const Default: Story = {
   args: {
```

Comment:
Not the right line, but: maybe we should add `default` to the controls? You can't easily go back to the initial state using controls now.

Reply:
For `severity`, yes. Added it.